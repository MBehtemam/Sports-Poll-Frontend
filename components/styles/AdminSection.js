import styled from "styled-components";

const AdminSection = styled.section`
  display: flex;
  justify-content: center;
`;
export default AdminSection;
