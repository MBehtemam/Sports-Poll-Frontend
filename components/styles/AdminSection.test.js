import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import AdminSection from "./AdminSection";

describe("Test suits for Admins section", () => {
  it("should match with snapshot", () => {
    const tree = renderer.create(<AdminSection />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("should has correct styled rule", () => {
    const tree = renderer.create(<AdminSection />).toJSON();
    expect(tree).toHaveStyleRule("display", "flex");
  });
});
