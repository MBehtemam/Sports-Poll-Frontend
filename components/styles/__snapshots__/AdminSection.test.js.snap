// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Admins section should match with snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<section
  className="c0"
/>
`;
